lib/mdl.rb

```diff
@@ -127,11 +133,28 @@ def self.run(argv = ARGV)
           docs_to_print << rule
         end
       end
+
+      next unless Config[:checkstyle]
+
+      results << add_errors_to_file_name_xml_node(
+        filename,
+        results_by_filename.join,
+      )
```

Comment:
I'd prefer this be wrapped in an `if`. That way someone doesn't accidentally add something below it.

```suggestion
      if Config[:checkstyle]
        results << add_errors_to_file_name_xml_node(
          filename,
          results_by_filename.join,
        )
      end
```